### ISSUE TITLE
[Feature request] Implement autocomplete for multiple tables without aliases

### ISSUE THREAD

**Issue by SylwesterZarebski** [dbeaver/dbeaver#7261]
### Problem
When using multiple joined tables, autocomplete for second and further tables is not available in main autocomplete suggestion list, only first table's fields are shown.

E.g.:
```
SELECT *
FROM table1 INNER JOIN table2 ON field1 = field2
```

When You try to use autocomplete instead of asterisk (not *on* asterisk, which expands asterisk to fields of table1) or inside `ON` clause (when You type <kbd>f</kbd>), fields are listed only for first table.

### Solution
When using autocomplete, list of fields should show all available fields from all joined tables.
Duplication of fields should not be a concern, but it could be next feature to automatic insert table or alias for duplicated fields.

### Alternatives
One way to use autocomplete is allowing to use relation between tables to create comparison for `ON` clause (using <kbd>CTRL+SPACE</kbd> after `ON`). It is not always sufficient when there are multiple relations.

Another way to use autocomplete for second and further tables is using aliases or full table with dot notation. It could be easier to use when implemented #7191.

E.g.:
```
SELECT *
FROM table1 INNER JOIN table2 ON field1 = table2.field2
```
Entering `table2.` then using autocomplete shows correctly fields for `table2`.

---
This issue is copy of https://github.com/dbeaver/dbeaver/issues/2230#issuecomment-333371082.

**Comment by tibuurcio** [comment]
For me the autocomplete on JOIN statements used to work, but I have since started to work on Mac OSX and it's not working anymore. Version 6.3.4

**Comment by uslss** [comment]
will be fixed in #12335

**Comment by SylwesterZarebski** [comment]
Thanks a lot!
